soccer/planning/RRTPlanner.cpp

```diff
@@ -157,6 +158,7 @@ std::unique_ptr<InterpolatedPath> RRTPlanner::generateRRTPath(
         // Check if Planning or optimization failed
         if (points.size() < 2) {
             debugLog("RRTPlanning Failed");
+            exit(1);
```

Comment:
Oops, I put that in for debugging purposes and forgot to take it out. Good catch :P